server/tsuserver.py

```diff
@@ -522,6 +546,60 @@ def send_discord_chat(self, name, message, hub_id=0, area_id=0):
             pos=self.config["bridgebot"]["pos"],
         )
 
+    def get_ffmpeg(self):
```

Comment:
It requires it so that it can extract the audio stream from the video downloaded. 
I am not proud of this function myself, but it felt like the ideal solution. Perhaps having the user acquire the binaries themselves would be better than doing this nonsense given the ffmpeg_location key, though I myself am not sure which is the better option of the two.